content/docs/deployment/helm/drivers/installation/powermax.md

```diff
@@ -366,11 +392,11 @@ CRDs should be configured during replication prepare stage with repctl as descri
 | vCenterCredSecret                  | Secret name for the vCenter credentials.                                                                                                                                                                                                                                                                                                                                        |  Yes      |   ""   |
 
 
-8. Install the driver using `csi-install.sh` bash script by running
+9. Install the driver using `csi-install.sh` bash script by running
     ```bash
     cd ../dell-csi-helm-installer && ./csi-install.sh --namespace powermax --values ./my-powermax-settings.yaml --helm-charts-version <version>
     ```
-9. Or you can also install the driver using standalone helm chart using the command
+10. Or you can also install the driver using standalone helm chart using the command
```

Comment:
Should an 'alternative method' for step 9 be its own step 10? Or should it be a sub-bullet or even the same level?

Reply:
It was previously another bullet I just increased the bullet number. Can merge to previous one if preferred.